packages/twenty-front/src/modules/views/components/EditableFilterDropdownButton.tsx

```diff
@@ -61,6 +61,13 @@ export const EditableFilterDropdownButton = ({
     removeViewFilter(viewFilter.fieldMetadataId);
   };
 
+  const onClickOutside = useCallback(() => {
```

Comment:
Our handlers are always named with handleXYZ, so in this case it would be handleDropdownClickOutside.